src/viur/core/__init__.py

```diff
@@ -143,10 +143,17 @@ def __build_app(modules: ModuleType | object, renderers: ModuleType | object, de
     from viur.core.modules.translation import Translation  # noqa: E402 # import works only here because circular imports
     from viur.core.prototypes.instanced_module import InstancedModule  # noqa: E402 # import works only here because circular imports
 
-    modules._tasks = TaskHandler
-    modules._moduleconf = ModuleConf
-    modules._translation = Translation
-    modules.script = Script
+    for name, cls in {
+        "_tasks": TaskHandler,
+        "_moduleconf": ModuleConf,
+        "_translation": Translation,
+        "script": Script,
+    }.items():
+        # Check whether name is contained in modules so that it can be overwritten
+        if name not in vars(modules):
+            setattr(modules, name, cls)
+
+        assert issubclass(getattr(modules, name), Module)
```

Comment:
We could enforce subclasses only
```suggestion
        assert issubclass(getattr(modules, name), cls)
```